.github/.wordlist.txt

```diff
@@ -88,6 +88,8 @@ armv
 ASAN
 asdk
 AssertionError
+ASR
+asr
```

Comment:
word lis tis not case sensitive I believe. Can keep only one.

Reply:
To do this, you will need to define a builder like the ones in https://github.com/project-chip/connectedhomeip/tree/master/scripts/build/builders

And then define targets in https://github.com/project-chip/connectedhomeip/blob/master/scripts/build/build/targets.py

Quite a few examples, generally to note:
  - make sure you do *not* use python functionality directly but rather emit the corresponding build shell commands. the intent is to be able to `--dry-run` and figure out what commands one would run manually to create a build
  - add any variants as needed (like debug options or enabling features) if that applies to you. no variants is also ok.

Reply:
@andy31415 update build_examples.py support

Reply:
@andy31415 hi, Andrei， can this PR be merged？